ghost/core/test/e2e-webhooks/pages.test.js

```diff
@@ -23,6 +24,17 @@ const tierSnapshot = {
 
 const roleSnapshot = tierSnapshot;
 
+const tagSnapshot = {
```

Comment:
The tag's name, slug, visibility etc. are not changing between test runs? If they are not, would be best to have snapshot capture them as is - no need to include non-dynamic properties in the snapshot matcher object 👍